battles/helpers/fight.py

```diff
@@ -0,0 +1,58 @@
+# rounds figth
+def compare_hitpoints(pokemon_1, pokemon_2):
+    if pokemon_1.hitpoints > pokemon_2.hitpoints:
+        return pokemon_1
+    return pokemon_2
+
+
+def compare_attack_defense(pokemon_1, pokemon_2):
+    if pokemon_1.attack > pokemon_2.defense:
+        return pokemon_1
+    return pokemon_2
+
+
+def get_pokemon_round_winner(pokemon_1, pokemon_2):
+    if compare_attack_defense(pokemon_1, pokemon_2) == compare_attack_defense(pokemon_2, pokemon_1):
+        return compare_attack_defense(pokemon_1, pokemon_2)
+    return compare_hitpoints(pokemon_1, pokemon_2)
+
+
+# battle fight
+def get_battle_winner(battle):
+    trainer_team_creator = battle.trainer_creator.teams.filter(battle_related=battle).first()
+    trainer_team_opponent = battle.trainer_opponent.teams.filter(battle_related=battle).first()
+    pokemons_creator = [
+        trainer_team_creator.pokemon_1,
+        trainer_team_creator.pokemon_2,
+        trainer_team_creator.pokemon_3
+    ]
+    pokemons_opponent = [
+        trainer_team_opponent.pokemon_1,
+        trainer_team_opponent.pokemon_2,
+        trainer_team_opponent.pokemon_3,
+    ]
+    rounds_trainers_winners = []
+
+    for (pokemon_c, pokemon_o) in zip(pokemons_creator, pokemons_opponent):
+        if get_pokemon_round_winner(pokemon_c, pokemon_o) == pokemon_c:
+            rounds_trainers_winners.append('creator')
+        else:
+            rounds_trainers_winners.append('opponent')
+
+    if rounds_trainers_winners.count('creator') > rounds_trainers_winners.count('opponent'):
+        return battle.trainer_creator
+    return battle.trainer_opponent
+
+
+def run_battle(battle):
+    battle.status = 'SETTLED'
+    battle.trainer_winner = get_battle_winner(battle)
+    return battle.save()
+
+
+def order_battle_pokemons(cleaned_data):
+    pokemons = [0, 1, 2]
```

Comment:
I think you missed fixing this.

Reply:
I commented above when you first pointed it https://github.com/pamella/pokebattle/pull/3#discussion_r266868218